import_flow_events.py

```diff
@@ -54,22 +59,63 @@
       utmTerm VARCHAR(40)
     );
 """
+Q_CREATE_METADATA_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_metadata (
+      flowId VARCHAR(64) NOT NULL UNIQUE SORTKEY,
+      beginTime TIMESTAMP NOT NULL,
+      duration INTERVAL NOT NULL,
+      completed BOOLEAN NOT NULL,
+      newAccount BOOLEAN NOT NULL,
+      browser VARCHAR(40),
+      browserVersion VARCHAR(40),
+      os VARCHAR(40),
+      context VARCHAR(40),
+      entrypoint VARCHAR(40),
+      migration VARCHAR(40),
+      service VARCHAR(40),
+      utmCampaign VARCHAR(40),
+      utmContents VARCHAR(40),
+      utmMedium VARCHAR(40),
+      utmSource VARCHAR(40),
+      utmTerm VARCHAR(40)
+    );
+"""
+Q_CREATE_EVENTS_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_events (
+      timestamp TIMESTAMP NOT NULL SORTKEY,
```

Comment:
I don't recall whether redshift properly processes foreign keys and `ON DELETE CASCADE` anyways.  I do know that it has problems handling unique key constraints (and by "problems" I mean "it just ignores them and lets you insert duplicates").  So we're probably better off doing own own explicit management of these things.